bolides.js

```diff
@@ -467,13 +549,14 @@ var bolides = {
         } else if (bolides.asteroid3.y <= -50) {
             bolides.asteroid3.isInMotion = false;
         }
+        /*
         // Collision detection
```

Comment:
There are actually two seperate functions because the spaceship and the bullets are different sizes.

Also, the main problem with the collision detection is that it doesn't take into account rotation of the spaceship.